pkg/apis/admissionregistration/types.go

```diff
@@ -160,7 +160,12 @@ type ValidatingAdmissionPolicySpec struct {
 	// +optional
 	Validations []Validation
 
-	// TODO align this with the webhook PR: https://github.com/kubernetes/kubernetes/pull/116261
+	// MatchConditions is a list of conditions on the AdmissionRequest that must be met
+	// for a request to be sent to be evaluated. All conditions must be met for the request to be matched.
+	// An empty list of matchConditions matches all requests. If there is an error evaluating the condition,
+	// the error is ignored and the condition is considered matched.
+	//
+	// +optional
```

Comment:
Mirror the API doc from https://github.com/kubernetes/kubernetes/blob/c19c57ed5348517e8063ba1ea91cf6adc73765c1/pkg/apis/admissionregistration/types.go#L744-L759

Reply:
ack